includes/general.php

```diff
@@ -922,19 +922,25 @@ function pods_shortcode_run( $tags, $content = null ) {
 		// id > slug (if both exist)
 		$id = null;
 
+		$evaluate_tags_args = array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 7b3c4d1783b6ec1fa06ca2b46f8c79a0</sub>